src/Orleans.Clustering.CosmosDB/Orleans.Clustering.CosmosDB.csproj

```diff
@@ -62,7 +62,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="1.9.1" />
+    <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="[1.9.1,2.0)" />
```

Comment:
Unrelated to this PR. It should be somewhere else. Please revert it.

Reply:
Will reply once for all three.  I do agree with this comment, and the changes are made in all three locations.  I will make a separate branch to contribute the nuget change(s) required, or please do feel free do so as you see fit.

Reply:
Seeing what I am getting at, I will revoke this PR if you are willing to make the necessary changes.  I've been trying to sort out the necessary changes, but they're not quickly clicking.  There are a few edge cases that seem to need additional thought and/or deeper knowledge of how the storage mechanism works, which I do not have.

Reply:
If you fix the tests failing I'm ok merging it. I'm just saying the package upgrade here should be a different PR and as Orleans 2.2 is about to be released, I would hold on with that change because I'll make the Orleans upgrade as soon as it is out.